pkg/util/resource_test.go

```diff
@@ -371,3 +371,70 @@ func TestQuantityPtr(t *testing.T) {
 		})
 	}
 }
+
+func TestLessThanOrEqualEnhanced(t *testing.T) {
+	tests := []struct {
+		name   string
+		a      corev1.ResourceList
+		b      corev1.ResourceList
+		expect bool
+	}{
+		{
+			name: "is equal",
+			a: corev1.ResourceList{
+				corev1.ResourceCPU:    *resource.NewQuantity(10, resource.DecimalSI),
+				corev1.ResourceMemory: *resource.NewQuantity(1024, resource.BinarySI),
+			},
+			b: corev1.ResourceList{
+				corev1.ResourceCPU:    *resource.NewQuantity(10, resource.DecimalSI),
+				corev1.ResourceMemory: *resource.NewQuantity(1024, resource.BinarySI),
+			},
+			expect: true,
+		},
+		{
+			name: "is equal",
```

Comment:
same case name with case1.
And missed case a > b.

Reply:
done